components/cdc/src/delegate.rs

```diff
@@ -507,40 +508,27 @@ impl Delegate {
         statistics: &mut Statistics,
         is_one_pc: bool,
     ) -> Result<()> {
-        debug_assert_eq!(self.txn_extra_op.load(), TxnExtraOp::ReadOldValue);
-        let mut read_old_value = |row: &mut EventRow, read_old_ts| -> Result<()> {
-            let key = Key::from_raw(&row.key).append_ts(row.start_ts.into());
-            let old_value = old_value_cb(key, read_old_ts, old_value_cache, statistics)?;
-            row.old_value = old_value.unwrap_or_default();
-            Ok(())
+        let txn_extra_op = self.txn_extra_op;
+
+        let entries = if txn_extra_op == TxnExtraOp::RawKv {
```

Comment:
There should be not `TxnExtraOp::RawKv` now. Need modify cdcpb.ChangeDataRequest, or just set a constant here.